RecoTauTag/HLTProducers/interface/L1TJetsMatching.h

```diff
@@ -197,10 +197,10 @@ class L1TJetsMatching: public edm::global::EDProducer<> {
      edm::ParameterSetDescription desc;
      desc.add<edm::InputTag>("L1JetTrigger", edm::InputTag("hltL1DiJetVBF"))->setComment("Name of trigger filter"    );
      desc.add<edm::InputTag>("JetSrc"      , edm::InputTag("hltAK4PFJetsTightIDCorrected"))->setComment("Input collection of PFJets");
-     desc.add<double>       ("pt1Min",95.0)->setComment("Minimal pT1 of PFJets to match");
-     desc.add<double>       ("pt2Min",35.0)->setComment("Minimal pT2 of PFJets to match");
-     desc.add<double>       ("mjjMin",650.0)->setComment("Minimal mjj of matched PFjets");
-     desc.add<double>       ("matchingR",0.5)->setComment("dR value used for matching");
+     desc.add<double>       ("pt1Min_",95.0)->setComment("Minimal pT1 of PFJets to match");
+     desc.add<double>       ("pt2Min_",35.0)->setComment("Minimal pT2 of PFJets to match");
+     desc.add<double>       ("mjjMin_",650.0)->setComment("Minimal mjj of matched PFjets");
+     desc.add<double>       ("matchingR_",0.5)->setComment("dR value used for matching");
```

Comment:
No underscore here: these are the variable names used in python config, typically there is NO trailing underscore
(as already ok for JetSrc)